SF:lib/core/errors/custom_exception.dart
DA:4,0
LF:1
LH:0
end_of_record
SF:lib/core/errors/failure.dart
DA:4,0
LF:1
LH:0
end_of_record
SF:lib/core/usecases/usecase.dart
DA:9,0
LF:1
LH:0
end_of_record
SF:lib/features/iot_unity_platform/domain/entities/iot_unity_platform_entity.dart
DA:6,2
DA:14,1
DA:15,1
DA:16,1
DA:17,1
LF:5
LH:5
end_of_record
SF:lib/features/iot_unity_platform/domain/repositories/iot_unity_platform_repository.dart
DA:8,0
LF:1
LH:0
end_of_record
SF:lib/features/iot_unity_platform/domain/usecases/get_data_from_iot_unity_platform_use_case.dart
DA:11,0
DA:17,0
DA:21,0
LF:3
LH:0
end_of_record
SF:lib/features/iot_unity_platform/data/models/iot_unity_platform_model.dart
DA:10,2
DA:15,1
DA:16,1
DA:18,0
LF:4
LH:3
end_of_record
SF:lib/features/iot_unity_platform/data/models/iot_unity_platform_model.g.dart
DA:9,1
DA:11,1
DA:12,2
DA:13,2
DA:16,0
DA:18,0
DA:19,0
DA:20,0
LF:8
LH:4
end_of_record
